modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -5,18 +5,18 @@
 
 :_content-type: PROCEDURE
 [id="nw-ingress-configuring-application-domain_{context}"]
-= Configuring application domain for the Ingress Controller Operator on AWS
+= Configuring application domain for the Ingress Controller Operator
 
 //OpenShift Dedicated or Amazon RH OpenShift cluster administrator
 
-As a cluster administrator, you can specify an alternative to the default cluster domain for user-created routes by configuring the `appsDomain` field. The `appsDomain` field is an optional domain for AWS infrastructure to use instead of the default, which is specified in the `domain` field. If you specify an alternative domain, it overrides the default cluster domain for the purpose of determining the default host for a new route.
+As a cluster administrator, you can specify an alternative to the default cluster domain for user-created routes by configuring the `appsDomain` field. The `appsDomain` field is an optional domain for cloud platforms to use instead of the default, which is specified in the `domain` field. If you specify an alternative domain, it overrides the default cluster domain for the purpose of determining the default host for a new route.
```

Comment:
```suggestion
As a cluster administrator, you can specify an alternative to the default cluster domain for user-created routes by configuring the `appsDomain` field. The `appsDomain` field is an optional domain for {product-title} to use instead of the default, which is specified in the `domain` field. If you specify an alternative domain, it overrides the default cluster domain for the purpose of determining the default host for a new route.
```